### ISSUE TITLE
Standardize buttons

### ISSUE THREAD

**Issue by adam-little** [cagov/design-system#662]
We don't have a great system for deciding how to pick the color of a button. We're also lacking consistency for how hover behaviors are applied.

- [x] Mockup styles in Figma
- [x] Start a component documentation page [Update: Documentation is [in Notion here](https://www.notion.so/odi-engineering/Button-b628202147124395a3a2bb5ad6affa10)]
- [x] Document rules and expected behaviors
- [ ] Establish naming convention (@chachasikes)
- [x] Update CSS (@kkoryaka)
- [x] Fix HTML implementation (@xjensen)
- [x] Update individual components that use buttons

**Comment by adam-little** [comment]
Feedback from @kkoryaka: in figma could you please create button sizes variations?

**Comment by m-sullivan7** [comment]
I removed myself from this issue since we're not ready for component documentation yet.

**Comment by chachasikes** [comment]
How is this going?

**Comment by adam-little** [comment]
@kkoryaka 

I documented the button states in Figma: 

- https://www.figma.com/file/veQuKQHf8gK5JdFEoMLGRD/UI-Design-kit?node-id=431%3A4658

Here are where buttons are used on the Design System website:

- [ ] Hero card
- [ ] Principle card (homepage)
- [ ] Contact form (homepage)
- [ ] Who's use the design system cards (homepage)
- [ ] Page feedback form
- [ ] Contact form (contact us)

Here's a summary of updates:

- Increased padding on left and right side of buttons (8px 16px 8px 16px)
- Reduced corner radius to 4px
- Increased button text weight to Public Sans Semibold
- Hover state: removed underline of text label
- Hover state: added dropshadow to button
- Focus state: Same properties as hover state + highlight border
- Active state: Same properties as hover state minus dropshadow

**Comment by yesenia-gtz** [comment]
@adam-little  @kkoryaka 
Hello, checking in on this Will these updates make it into the next release? I am gathering release updates for DWSN share out on April 21
